tests/hikari/impl/test_rest.py

```diff
@@ -817,8 +911,12 @@ class StubResponse:
         rest_client._debug = False
         rest_client._parse_ratelimits = mock.AsyncMock()
         rest_client._acquire_client_session = mock.Mock(return_value=mock_session)
-        with mock.patch.object(asyncio, "gather", new=mock.AsyncMock()):
-            assert (await rest_client._request(route)) is None
+        rest_client._stringify_http_message = mock.Mock()
+        with mock.patch.object(rest, "_LOGGER", new=mock.Mock(getEffectiveLevel=mock.Mock(return_value=100))) as logger:
+            with mock.patch.object(asyncio, "gather", new=mock.AsyncMock()):
+                assert (await rest_client._request(route)) is None
+
+        assert logger.log.call_count == 0
```

Comment:
assert_not_called?

Reply:
btw just a random point but if you're testing logging there are other places where that's being done in the repo already encase you want to change how it's being tested in either place for consistency https://github.com/nekokatt/hikari/blob/69a9ac65ddb1a79022508ab592415f336f23d00d/tests/hikari/utilities/test_event_stream.py#L203

Reply:
This doesnt specifically test logging, its to follow some paths on the code. Im going to change this to its own test